templates/CRM/Admin/Page/Extensions/Main.tpl

```diff
@@ -21,10 +20,11 @@ Depends: CRM/common/enableDisableApi.tpl and CRM/common/jsortable.tpl
           <td class="crm-extensions-label">
             <details class="crm-accordion-light">
               <summary>
-              <strong>{$row.label|escape}</strong><br/>{$row.description|escape}
-              {if $extAddNewEnabled && array_key_exists($extKey, $remoteExtensionRows) && $remoteExtensionRows[$extKey].upgradelink|smarty:nodefaults}
-                <div class="crm-extensions-upgrade">{$remoteExtensionRows[$extKey].upgradelink|smarty:nodefaults}</div>
-              {/if}
+                <strong>{$row.label|escape}</strong>
+                <br/>{$row.description|escape}
+                {if $extAddNewEnabled && array_key_exists($extKey, $remoteExtensionRows) && $remoteExtensionRows[$extKey].upgradelink|smarty:nodefaults}
+                  <div class="crm-extensions-upgrade">{$remoteExtensionRows[$extKey].upgradelink|smarty:nodefaults}</div>
+                {/if}
```

Comment:
This is just reformatting so that it's the same as in AddNew.tpl. I was tempted to merge both templates, which are 95% the same, but it's probably not worth it because we will replace by SearchKit.